### ISSUE TITLE
argparse help error: arguments created by add_mutually_exclusive_group() are shown outside their parent group created by add_argument_group()

### ISSUE THREAD

**Issue by b0f77e89-8cff-40ac-804c-f3be88dfec68** [python/cpython#70070]
BPO | [25882](https://bugs.python.org/issue25882)
--- | :---
Files | <li>[issue25882.py](https://bugs.python.org/file41332/issue25882.py "Uploaded as text/plain at 2015-12-17.03:32:00 by paul.j3")</li>

<sup>*Note: these values reflect the state of the issue at the time it was migrated and might not reflect the current state.*</sup>

<details><summary>Show more details</summary><p>

GitHub fields:
```python
assignee = None
closed_at = None
created_at = <Date 2015-12-16.13:14:43.143>
labels = ['type-bug']
title = 'argparse help error: arguments created by add_mutually_exclusive_group() are shown outside their parent group created by add_argument_group()'
updated_at = <Date 2017-02-26.18:04:42.671>
user = 'https://bugs.python.org/balage'
```

bugs.python.org fields:
```python
activity = <Date 2017-02-26.18:04:42.671>
actor = 'paul.j3'
assignee = 'none'
closed = False
closed_date = None
closer = None
components = []
creation = <Date 2015-12-16.13:14:43.143>
creator = 'balage'
dependencies = []
files = ['41332']
hgrepos = []
issue_num = 25882
keywords = []
message_count = 7.0
messages = ['256518', '256569', '256573', '256597', '256598', '256612', '288606']
nosy_count = 3.0
nosy_names = ['paul.j3', 'srikanth', 'balage']
pr_nums = []
priority = 'normal'
resolution = None
stage = None
status = 'open'
superseder = None
type = 'behavior'
url = 'https://bugs.python.org/issue25882'
versions = ['Python 2.7']
```

</p></details>

**Comment by 7a064fe6-c535-4d80-a11f-a04ed39056c5** [comment]
Earlier issue on the same topic - passing a mutually exclusive group via parents

http://bugs.python.org/issue16807

Can they be consolidated?

**Comment by serhiy-storchaka** [comment]
Closed as a duplicate of #61011.
